feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio.c

```diff
@@ -814,13 +814,38 @@ static int conn_since = 0;
 
 static void apc_enable(bool flag) {
 
-	SCHEMA_SET_INT(apc_conf.enabled, flag);
-	if (!ovsdb_table_update(&table_APC_Config, &apc_conf)) {
-		LOG(ERR, "%s:APC_Config: failed to update", __func__);
-		return;
-	}
+	struct schema_APC_State apc_state;
+	json_t *where;
+	
 	LOGI("APC %s: %s APC", __func__, flag?"enable":"disable");
+	if (flag == false) {
+		where = ovsdb_table_where(&table_APC_State, &apc_state);
+		if (false == ovsdb_table_select_one_where(&table_APC_State,
```

Comment:
will it be good to make  **flag** == ovsdb_table_select_one_where(&amp;table_APC_State, where, &amp;apc_state)), since flag will be always false within this if condition.

Reply:
That flag represents if the APC is to be enabled/disabled, so it would be confusing if we use it to check return value of that function.